roles/byowl/tasks/main.yml

```diff
@@ -6,3 +6,23 @@
     state: present
     definition: "{{ lookup('template', 'workload.yml') | from_yaml }}"
   when: byowl.kind is not defined
+
+- name: Wait for fiod job to succeed...
+  k8s_facts:
+    kind: Job
+    api_version: batch/v1
+    name: '{{ meta.name }}-byowl'
+    namespace: "{{ operator_namespace }}"
+  register: byowl_jobs
+  until: "byowl_jobs | json_query('resources[].status.succeeded')"
+  retries: 1000
+  delay: 20
+
+- k8s_status:
```

Comment:
Should this have a `when` condition based on the success of the 'wait for job to succeed' task?

Reply:
`Completed` does not necessarily mean `Success`.